docs/timelion/getting-started/timelion-create.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[timelion-create]]
+=== Creating time series visualizations
+
+This tutorial will be using the time series data from https://www.elastic.co/guide/en/beats/metricbeat/current/index.html[Metricbeat] to walk you through a number of functions that Timelion offers. To get started, download Metricbeat and follow the https://www.elastic.co/downloads/beats/metricbeat[instructions here] to start ingesting the data locally.
+
+The first visualization you will create compares the average percentage of CPU time spent in user space to the previous hours. In order to create this visualization, we’ll need to create two Timelion expressions. One with the current average of `system.cpu.user.pct` and another with the average from the previous hour.
```

Comment:
>The first visualization you will create compares the average percentage of CPU time spent in user space to the previous hours.

This sentence is not clear. What's the bucket interval for the average percentage of CPU time spent in user space that is being compared to that of the previous hours? 

>One with the current average of `system.cpu.user.pct` and another with the average from the previous hour.

This sentence says "previous hour". I'm not sure which time bucket we are comparing.